packages/model-viewer/src/features/scene-graph/pbr-metallic-roughness.ts

```diff
@@ -56,49 +56,58 @@ export class PBRMetallicRoughness extends ThreeDOMElement implements
       pbrMetallicRoughness.metallicFactor = 1;
     }
 
-    const baseColorTexture =
-        pbrMetallicRoughness.baseColorTexture ?? {index: -1};
+    let {
+      baseColorTexture: gltfBaseColorTexture,
+      metallicRoughnessTexture: gltfMetallicRoughnessTexture
+    } = pbrMetallicRoughness;
 
-    const metallicRoughnessTexture =
```

Comment:
refactored this code to make it look consistent with the way things are done in material